packages/fluentui/docs/src/examples/components/Menu/Visual/MenuExampleCompose.shorthand.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import {
+  Menu,
+  MenuItem,
+  compose,
+  Provider,
+  MenuItemIcon,
+  MenuItemContent,
+  MenuItemIndicator,
+  MenuItemWrapper,
+  BookmarkIcon,
+} from '@fluentui/react-northstar';
+
+const MenuItemWrapperDashed = compose(MenuItemWrapper, {
+  displayName: 'MenuItemWrapperDashed',
+});
+
+const MenuItemIconGreen = compose(MenuItemIcon, {
+  displayName: 'MenuItemIconGreen',
+});
+
+const MenuItemContentPink = compose(MenuItemContent, {
+  displayName: 'MenuItemContentPink',
+});
+
+const MenuitemIndicatorSaturated = compose(MenuItemIndicator, {
+  displayName: 'MenuitemIndicatorSaturated',
+});
+
+const MenuItemBlue = compose(MenuItem, {
+  displayName: 'MenuItemBlue',
+  slots: {
+    icon: MenuItemIconGreen,
+    content: MenuItemContentPink,
+    indicator: MenuitemIndicatorSaturated,
+    wrapper: MenuItemWrapperDashed,
+  },
+});
+
+const items = [
+  {
+    key: 'editorials',
+    content: 'Editorials',
+    icon: <BookmarkIcon />,
+    menu: ['One', 'Two', 'Three'],
+    children: (C, p) => <MenuItemBlue {...p} />,
```

Comment:
```suggestion
    children: (C, p) => <MenuItemBlue {...p} />,
```

Should we add a todo there?